addons/core/translations/resources/en-us.yaml

```diff
@@ -45,6 +45,7 @@ auth-method:
     delete: Delete Auth Method
     make-primary: Make Primary
     remove-as-primary: Remove as Primary
+    change-state: State
```

Comment:
Not sure this key->value mapping makes sense.

Reply:
Refactored this to `status: Status` to better represent auth-method status.